templates/zerver/api/changelog.md

```diff
@@ -20,6 +20,13 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 6.0
 
+**Feature level 151**
+
+* [`POST /register`](/api/register-queue), [`GET /events`](/api/get-events),
+  [`POST /realm/profile_fields`](/api/create-custom-profile-field),
+  [`GET /realm/profile_fields`](/api/get-custom-profile-fields): Added
+  `PRONOUNS` custom profile field type.
```

Comment:
Does the string `PRONOUNS` actually appear in the API itself, or just in the `models.py` labels for these? I wonder if we should just have this say "Added pronouns custom profile field type." here, and discuss the field type 9 instead of "PRONOUNS" elsewhere in the API docs.